guide.md

```diff
@@ -13492,6 +13492,7 @@ See the [Some last OPSEC thoughts][Some last OPSEC thoughts:] section for some t
   [Note about Plausible Deniability:]: #note-about-plausible-deniability
   [Installation:]: #installation-3
   [Lid Closure Behavior:]: #lid-closure-behavior
+  [Anti Evil Maid (AEM):]: #anti-evil-maid-aem

```

Comment:
We should remember to add sections here when adding sections in the guide. Just a small oversight. Not a big deal.